db/schema.rb

```diff
@@ -477,7 +477,6 @@
     t.datetime "updated_at", null: false
     t.bigint "user_id"
     t.datetime "verified_at"
-    t.index ["user_id"], name: "index_email_authorizations_on_user_id"
```

Comment:
I think this file was included by accident 🙂

Reply:
ugh, yea.. i'll revert that one!